src/telemetryReporter.ts

```diff
@@ -201,12 +199,12 @@ export default class TelemetryReporter {
                 }
             }
 
-            const nodeModulesRegex = /^[\\\/]?(node_modules|node_modules\.asar)[\\\/]/;
-            const fileRegex = /(file:\/\/)?([a-zA-Z]:(\\\\|\\|\/)|(\\\\|\\|\/))?([\w-\._]+(\\\\|\\|\/))+[\w-\._]*/g;
+            const nodeModulesRegex = /^[\\/]?(node_modules|node_modules\.asar)[\\/]/;
+            const fileRegex = /(file:\/\/)?([a-zA-Z]:(\\\\|\\|\/)|(\\\\|\\|\/))?([\w-._]+(\\\\|\\|\/))+[\w-._]*/g;

```

Comment:
remove the `\` before '.' will make it match any single character

Reply:
I think the behaviour is a little different because it is in square brackets
![image](https://user-images.githubusercontent.com/1297909/101927026-00420700-3ba2-11eb-800f-2d1daa10416b.png)
That's why it is OK with the escaped period on the first line, but doesn't like them for the second.
The docs are a little light for that rule https://eslint.org/docs/rules/no-useless-escape, but it's actually suggesting in the code that the `-` might need to be escaped in the second line.
If you have a sample file pattern I can play with it a bit before just disabling the rule for the line